contributing.md

```diff
@@ -62,6 +63,13 @@ Code contributions are very welcome.  It’s often good to first create an issue
 to report a bug or suggest a new feature before creating a pull request to
 prevent you from doing unnecessary work.
 
+## Running the tests
+
+1.  `npm/yarn install`
```

Comment:
We should probably prefer one dep manager and update contrib docs to reflect that. Perhaps here we should prefer `yarn`?

```suggestion
1.  `yarn`
```